bin/test

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env bash
+# https://circleci.com/docs/2.0/using-shell-scripts/
+# set -o nounset
+set -o errexit
+set -o xtrace
+
+mix hex.audit
```

Comment:
Have you seen `dependabot`? https://dependabot.com/

It was acquired by GitHub a few months ago so will probably become part of the core offering in the future.

Reply:
Yes. I just added it to this repo.